app/templates/gulp/lint.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var gulp   = require('gulp');
+var plugins = require('gulp-load-plugins')();
+
+module.exports = function (paths, config) {
+  gulp.task('lint', function () {
+    return gulp.src(paths.lint)
+        .pipe(plugins.jshint('.jshintrc'))<% if (jscsModule) { %>
```

Comment:
Expected an identifier and instead saw '%'.<br>Unrecoverable syntax error. (60% scanned).